app/serializers/flight_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class FlightSerializer < ActiveModel::Serializer
+  attribute :fare_category
+
+  attribute :price do
+    object.price&.formatted_price
```

Comment:
Uma vez que um `Flight` tem apenas um `Price`, poderíamos usar `delegate` no model para evitar a necessidade de acessar `Flight#price#formatted_price` e, assim, observar a [Lei de Deméter](https://blog.testdouble.com/posts/2022-06-15-law-of-demeter/).

Reply:
Interessante, não conhecia sobre `delegate`. Ajustado em: https://github.com/Laryssa-lfa/flight-company/pull/5/commits/4f1bfecec6b8ddb81dd71807ea44c47d191c92a7.